packages/components/src/popover/index.js

```diff
@@ -483,6 +485,24 @@ const Popover = ( {
 
 const PopoverContainer = Popover;
 
-PopoverContainer.Slot = () => <Slot bubblesVirtually name={ SLOT_NAME } />;
```

Comment:
I'm not sure I understand why the change here and why we need multiple Popover Slots?

Reply:
`Popovers` should ideally be rendered in the nearest scrollable parent container so that the z-indices don't break. So Popovers with anchors in the content will render within the content scroll container, and Popovers in e.g. the header will render in the document scroll area. @jasmussen does this make sense to you?